### PR TITLE
[KHWEBA - 21]Edit account

### PR DESCRIPTION
**Pull request by OlgaOrlova**
image
Edit account

### CODE REVIEW

**Review by Vladyslav-Plakhuta — DISMISSED** [review]

File: src/components/add-account-dialog/add-account-dialog.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <aside class="add-account-dialog mdc-dialog" role="alertdialog">
   <div class="mdc-dialog__surface">
     <header class="mdc-dialog__header">
-      <h2 class="mdc-dialog__header__title">
+      <h2 class="add-account-dialog-title mdc-dialog__header__title">
```
Comment (gift-a) [inline src/components/add-account-dialog/add-account-dialog.html]: this dialog isn't "add dialog" anymore. it is universal "dialog"

**Comment by gift-a** [comment]
Do you change accounts info in transactions?

**Review by Vladyslav-Plakhuta — DISMISSED** [review]

File: src/components/add-account-dialog/add-account-dialog.html (hunk 1 above)
Comment (ihor7016) [inline src/components/add-account-dialog/add-account-dialog.html]: where is BEM?

File: src/components/accounts/accounts.js [hunk 2]
```diff
@@ -119,8 +132,14 @@ export class AccountsComponent {
     this.initMoreBtns();
   }
 
-  handleEditClick() {
-    console.log("handleEditClick");
+  handleEditClick(event) {
+    let moreButton = event.target.closest(".button-more");
```
Comment (ihor7016) [inline src/components/accounts/accounts.js]: please do not match classnames that are not in this component

File: src/components/add-account-dialog/add-account-dialog.js [hunk 3]
```diff
@@ -65,38 +84,56 @@ export class AddAccountDialogComponent {
...
-      name: this.accountNameInput.value,
-      balance: parseInt(this.balanceNameInput.value),
-      type: this.accountType.innerText,
-      currency: this.accountCurrency.innerText,
-      id: Math.random()
-        .toString(36)
-        .substring(2)
-    });
-    this.clean();
+    if (this.accountTitle.innerText === "Add account") {
```
Comment (ihor7016) [inline src/components/add-account-dialog/add-account-dialog.js]: you can set in `showDialogEdit ` method property on instance like `this.isEditMode` -- and here you will check it

**Review by ihor7016 — APPROVED** [review]

**Pull request merged**